spec/helpers/holding_block_spec.rb

```diff
@@ -134,7 +134,7 @@
         expect(show_result.last).to include library
       end
       it 'link to call number browse' do
-        expect(show_result.last).to have_link('[Browse]', href: "/browse/call_numbers?q=#{call_number}")
+        expect(show_result.last).to have_link('Browse nearby items', href: "/browse/call_numbers?q=#{call_number}")
```

Comment:
Update the spec to refer the locale value as well so changes to the label text don't breaking the spec.